admin_guide/scheduling/priority_preemption.adoc

```diff
@@ -88,51 +87,77 @@ spec:
 
-When Pods are created, they go to a queue and wait to be scheduled. The scheduler picks a Pod from the queue and tries to schedule it on a Node. If no Node is found that satisfies all the specified requirements of the Pod, preemption logic is triggered for the pending Pod. Let’s call the pending Pod P. Preemption logic tries to find a Node where removal of one or more Pods with lower priority than P would enable P to be scheduled on that Node. If such a Node is found, one or more lower priority Pods get deleted from the Node. After the Pods are gone, P can be scheduled on the Node.
+When the scheduler prempts one or more pods on a node, the `nominatedNodeName` field of higher-priority pod specification is set to the name of the node. The scheduler uses this field to keep track of the resources reserved for pods and also provides user information about preemptions in their clusters.
 
-=== User exposed information
+After the scheduler preempts a lower-priority pod, the scheduler honors the graceful termination period of the pod. If another node becomes available while scheduler is waiting for the lower-priority pod to terminate, the scheduler can schedule the higher-priority pod on the node. As a result, the `nominatedNodeName` field and `nodeName` field of the pod specification might be different. 
 
-When Pod P preempts one or more Pods on Node N, nominatedNodeName field of Pod P’s status is set to the name of Node N. This field helps scheduler track resources reserved for Pod P and also gives users information about preemptions in their clusters.
+Also, if the scheduler preempts pods on a node and is waiting for termination, if a pod with a higher-priority pod than the waiting pod needs to be scheduled, the scheduler can schedule the higher-priority pod instead. In such a case, the scheduler clears the `nominatedNodeName` of the waiting pod making the pod eligible for another node.
 
-Please note that Pod P is not necessarily scheduled to the “nominated Node”. After victim Pods are preempted, they get their graceful termination period. If another node becomes available while scheduler is waiting for the victim Pods to terminate, scheduler will use the other node to schedule Pod P. As a result nominatedNodeName and nodeName of Pod spec are not always the same. Also, if scheduler preempts Pods on Node N, but then a higher priority Pod than Pod P arrives, scheduler may give Node N to the new higher priority Pod. In such a case, scheduler clears nominatedNodeName of Pod P. By doing this, scheduler makes Pod P eligible to preempt Pods on another Node.
+Preemption does not necessarily remove all lower-priority pods. The scheduler can schedule a pending pod by removing a portion of the lower-priority pods are removed. 
```

Comment:
typo: `The scheduler can schedule a pending pod by removing a portion of the lower-priority pods are removed. `